src/Asset/Attributes/Property/UpdateCustomMetaData.php

```diff
@@ -28,9 +28,9 @@ final class UpdateCustomMetaData extends Property
     public function __construct()
     {
         parent::__construct(
-            'metadata',
+            property: 'metadata',
```

Comment:
should we also adapt the property names?

Reply:
I would not, since this is used by the serializer and I would not change the API property names?
Or should we?

Reply:
Not sure to be honest. For custom settings fro example we use customSettings ...so maybe just to keep it consistent?

Reply:
I would even prefer it the other way round to write everything small metadata, but that are also more changes, because it is actually one word.

Reply:
@lukmzig @martineiber should I turn it around and make it metadata everywhere instead of metaData since it is actually its own word?

Reply:
maybe that would make more sense?
since I guess in the core its also called metadata based on this:
https://github.com/pimcore/studio-backend-bundle/pull/313/files#diff-69c196c7cd38d69b327debc1f015c2d583aa9f9f20c753ca2839b74ed2b4aad7R34

Reply:
Resolved https://github.com/pimcore/studio-backend-bundle/pull/313/commits/cc45a2b9a08918fc423ada5cffd9cc2ffa89ecc0